styles/avatar/avatar.css

```diff
@@ -0,0 +1,42 @@
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+:root {
+  --primary-color: white;
+  --secondary-color: rgb(15, 1, 1);
+  --default: rgb(53, 52, 52);
+}
+
+.avatar-container{
+  padding: 1rem;
+}
+.avatar-round {
+  border-radius: 50%;
+}
+.avatar-square {
+  border-radius: 25%;
+}
+img{
```

Comment:
Just a suggestion , every img element will get margin of 1rem.

Reply:
thanks kevin, I will change that